vl-kontrakt-behandlingsprosess-vedtak/v2/src/main/resources/xsd/beregningsgrunnlag/beregningsgrunnlag-svp-v2.xsd

```diff
@@ -0,0 +1,180 @@
+					<xsd:element name="beregningsgrunnlagPeriode" type="bg:BeregningsgrunnlagPeriode" minOccurs="0" maxOccurs="unbounded"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:complexType name="BeregningsgrunnlagPeriode">
+		<xsd:sequence>
+			<xsd:element name="periode" type="felles:PeriodeOpplysning" minOccurs="0"/>
+			<xsd:element name="brutto" type="felles:FloatOpplysning" minOccurs="0"/>
+			<xsd:element name="avkortet" type="felles:FloatOpplysning" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Avkortet beregningsgrunnlag</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="redusert" type="felles:FloatOpplysning" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Redusert beregningsgrunnlag</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="dagsats" type="felles:LongOpplysning" minOccurs="0"/>
+			<xsd:element name="beregningsgrunnlagPrStatusOgAndel" type="bg:BeregningsgrunnlagPrStatusOgAndel" minOccurs="0" maxOccurs="unbounded"/>
+		</xsd:sequence>
+	</xsd:complexType>
+	<xsd:complexType name="BeregningsgrunnlagPrStatusOgAndel">
+		<xsd:sequence>
+			<xsd:element name="periode" type="felles:PeriodeOpplysning" minOccurs="0"/>
+			<xsd:element name="aktivitetstatus" type="felles:KodeverksOpplysning" minOccurs="0"/>
+			<xsd:element name="virksomhetsnummer" type="felles:StringOpplysning" minOccurs="0"/>
+			<xsd:element name="erTidsbegrensetArbeidsforhold" type="felles:BooleanOpplysning" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Angir om andelen angår et tidsbegrenset arbeidsforhold</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="brutto" type="felles:FloatOpplysning" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Brutto beregningsgrunnlagandel</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="avkortet" type="felles:FloatOpplysning" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Avkortet beregningsgrunnlagandel</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="redusert" type="felles:FloatOpplysning" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Redusert beregningsgrunnlagandel</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="overstyrt" type="felles:FloatOpplysning" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Beregningsgrunnlagandel overstyrt</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="beregnet" type="felles:FloatOpplysning" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Beregningsgrunnlagandel beregnet</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="inntektskategori" type="felles:KodeverksOpplysning" minOccurs="0">
```

Comment:
Burde referanser til kodeverk peke på hvilket kodeverk som benyttes? (som dokumentasjon, ikke inkludere det hvis det er kodeverk som naturlig endres

Reply:
inntektskategori er enum i fpsak . Har lagt på dette som komentar : "Internt kodeverk INNTEKTSKATEGORI som angir inntektskategori for andelen"